src/status_im/subs.cljs

```diff
@@ -863,7 +856,9 @@
  :chats/unread-messages-number
  :<- [:chats/active-chats]
  (fn [chats _]
-   (apply + (map :unviewed-messages-count (vals chats)))))
+   (let [grouped-chats (group-by :public? (vals chats))]
+     {:public (apply + (map :unviewed-messages-count (get grouped-chats true)))
```

Comment:
`(reduce + ...`? :)

Reply:
what the difference?

Reply:
It just looks more idiomatic to me to use `reduce` here. But I've just found that this is an old debate topic https://stackoverflow.com/questions/3153396/clojure-reduce-vs-apply , not a big deal

Reply:
on the second thought 
```clojure
(transduce (map :unviewed-messages-count) (completing +) (get grouped-chats true))
```
could do the same thing in one go, doesn't matter here though